.travis.yml

```diff
@@ -0,0 +1,24 @@
+# You must sign into travis-ci.org and set the commit hook on your project for travis to
+# run on your project. The secure: variable must be generated by running `travis encrypt`
+# on a github oauth key that you can generate using curl.
+
+sudo: false
+language: node_js
+node_js:
+  - 'stable'
+addons:
+  firefox: latest
```

Comment:
When I tried adding both, it seemed like the firefox beta install would override the firefox stable install (most likely because it was the last one defined).

Looking online everyone suggests that to have both at the same time on a linux machine, you need to save the firefox beta tar somewhere different to the stable rather than get it through a package manager.